### ISSUE TITLE
SSH log in requires password

### ISSUE THREAD

**Issue by fenollp** [timower/rM-docker#11]
Readme says:
```
# Now login using the root account, no password
# Or ssh:
ssh root@localhost -p 2222
```
but with 
```shell
ssh root@localhost -p 2222
```
a password is required.

Note: the password also changes at each new `docker run --rm -it ..`

**Comment by Eeems** [comment]
Did you run xochitl?

**Comment by timower** [comment]
Indeed, running xochitl will change the root password on first run. We can probably have the script log it, or somehow prevent xochitl from setting it

**Comment by fenollp** [comment]
Yes this is when I'm running `xochitl`. I'd be happy with solution that SSH-logins with my local/given via env SSH key.

**Comment by Eeems** [comment]
Perhaps setting up something similar to [sshpiper](https://github.com/tg123/sshpiper/) to handle the authentication from the container to qemu would be a good solution? That way you can setup a SSH key between the container and qemu, and then dynamically map in the authentication that you would use to connect from the host system to sshpiper as part of creating the container.

**Comment by timower** [comment]
xochitl calls `usermod -p` to set the password, we could also prevent that.

**Comment by FluffLescure** [comment]
this is dumb but will work,  you  could overwrite usermod. Besides it isn't all that usefull anyway

**Comment by FluffLescure** [comment]
Adding `ssh -o StrictHostKeyChecking=no root@localhost 'rm /usr/sbin/usermod' ` to Dockerfile at line 150 prevents xochitl from calling usermod, allowing ssh login without password.

**Comment by StarNumber12046** [comment]
that's probably not a good idea

**Comment by StarNumber12046** [comment]
just make it log it
